### PR TITLE
Merging kubernetesmastervarsvmss into kubernetesmastervars

### PR DESCRIPTION
**Pull request by tariq1890**
Merging the vmss template variants back into the original template files. This change is being done to remove redundancy.

### CODE REVIEW

**Comment by tariq1890** [comment]
@jackfrancis Ready for review

**Comment by tariq1890** [comment]
@jackfrancis @ritazh Requesting.eyes on this :)

File: parts/k8s/kubernetesmastervars.t [hunk 1]
```diff
@@ -236,6 +256,18 @@
     "masterFirstAddrOctets": "[split(parameters('firstConsecutiveStaticIP'),'.')]",
     "masterFirstAddrOctet4": "[variables('masterFirstAddrOctets')[3]]",
     "masterFirstAddrPrefix": "[concat(variables('masterFirstAddrOctets')[0],'.',variables('masterFirstAddrOctets')[1],'.',variables('masterFirstAddrOctets')[2],'.')]",
+    "masterEtcdServerPort": {{GetMasterEtcdServerPort}},
+    "masterEtcdClientPort": {{GetMasterEtcdClientPort}},
+    {{if IsMasterVirtualMachineScaleSets}}
+    "masterVMNamePrefix": "[concat(parameters('orchestratorName'), '-master-', parameters('nameSuffix'), '-')]",
+    "masterVMNames": [
```
Comment (ritazh) [inline parts/k8s/kubernetesmastervars.t]: I don't think `masterVMNames` is ever used by VMSS masters.

Comment (tariq1890) [reply parts/k8s/kubernetesmastervars.t]: This was added in Kubernetesmastersvmss. So I had it copied it over from that file. Do you mean that it is okay to remove?

Comment (ritazh) [reply parts/k8s/kubernetesmastervars.t]: Yes, I meant it's not being used currently. So we should remove it.

**Comment by tariq1890** [comment]
@ritazh Done :)

**Pull request merged**